### PR TITLE
dart errors on css/html tab

### LINKED ISSUE #98: dart errors show up on html, css tabs

### PR DESCRIPTION
**Pull request by kasperpeulen**
@devoncarew @lukechurch 
fix #98

### CODE REVIEW

File: lib/playground.dart [hunk 1]
```diff
@@ -537,6 +541,10 @@ class Playground implements GistContainer, GistController {
   /// Perform static analysis of the source code. Return whether the code
   /// analyzed cleanly (had no errors or warnings).
   Future<bool> _performAnalysis() {
+
+    // We only have issues for dart source code.
+    if (context.focusedEditor != "dart") return null;
```
Comment (devoncarew) [inline lib/playground.dart]: Is this necessary? Hopefully we're only calling `_performAnalysis()` when the dart text is mutated (or when the gist is first loaded).

File: lib/playground.dart [hunk 2]
```diff
@@ -432,6 +432,10 @@ class Playground implements GistContainer, GistController {
     component.onClick.listen((_) {
       if (component.hasAttr('selected')) return;
 
+      Element issuesElement = querySelector('#issues');
+      if (name != "dart") issuesElement.style.display = "none";
```
Comment (devoncarew) [inline lib/playground.dart]: Very elegant!

**Comment by devoncarew** [comment]
Kasper! Welcome back.

I have one question about whether a line is necessary for this fix. If so, lgtm!

I'll address the failing saucelabs/webdriver test separately - it's not related to this PR.

File: lib/playground.dart (hunk 1 above)
Comment (devoncarew) [inline lib/playground.dart]: And if it is necessary, we should change to returning `new Future.value(true)` (to tell clients the the code analyzed cleanly).

Comment (kasperpeulen) [inline lib/playground.dart]: oh wait, you are right, according to webstorm, `_performAnalysis()` is only called on `showHome`, when the gist is first loaded, and when the dart text is mutated.

So I'll remove this.

**Comment by kasperpeulen** [comment]
>Kasper! Welcome back.

thanks, glad to be back !

**Comment by kasperpeulen** [comment]
oh wait, stupid, I was just reordening some code (to make things clearer for myself) and those changes also went in, I will change that back @devoncarew

**Comment by kasperpeulen** [comment]
see #440

**Pull request closed without merging**